xenit/solr6/local/93-restore-from-backup.sh

```diff
@@ -85,7 +85,7 @@ set -e
 
 echo "Solr restore from backup start"
 
-if [ -z "${RESTORE_FROM_BACKUP}" ]; then
+if [ -z "${RESTORE_FROM_BACKUP}" ] || [ "${RESTORE_FROM_BACKUP}" != "true" ]; then
```

Comment:
You want to default to be to restore from backup. So use this condition: `if [ "${RESTORE_FROM_BACKUP}" = "false" ]; then` 
That means:
1. var absent? -> restore
2. var present ? -> restore
3. var present but it is explicitly `false`? -> do not restore

Reply:
Discussed... Default is better as "false" and only do it when explicitly set to "true".